### PR TITLE
Removing deprecated jaeger and jaegerthrifthttp exporters

### PR DESCRIPTION
**Pull request by codeboten**
This follows the deprecation plan to remove the component. The original removal date was July 2023, it is now past that.

Linked issue: https://github.com/open-telemetry/opentelemetry-specification/issues/3551

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

**Review by frzifus — APPROVED** [review]

File: .chloggen/codeboten_rm-jaeger-exporters.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: jaegerexporter, jaegerthrifthttpexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Removing deprecated jaeger and jaegerthrifthttp exporters
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: []
```
Comment (mx-psi) [inline .chloggen/codeboten_rm-jaeger-exporters.yaml:13]:
```suggestion
issues: [26546]
```

**Review by mx-psi — APPROVED** [review]

**Review by jpkrohling — APPROVED** [review]

File: .chloggen/codeboten_rm-jaeger-exporters.yaml (hunk 1 above)
Comment (codeboten) [inline .chloggen/codeboten_rm-jaeger-exporters.yaml:13]:
```suggestion
issues: [26546]
```

**Comment by bangbaew** [comment]
What should I do now when I previously use `otel/opentelemetry-collector-contrib` image as a gateway to export received spans to Jaeger and Zipkin, received logs to Loki, and export metrics to Prometheus so I can visualize spanmetrics in Jaeger UI SPM page, with jaeger exporter removed, I have to use jaeger port 4317, which is unable to achieve these features (or maybe it can but i don't know how).

On Jaeger's official tutorial to get SPM to work, they still use  `otel/opentelemetry-collector-contrib:0.80.0`, in their docker-compose.yml which was updated 3 days ago.

https://www.jaegertracing.io/docs/1.49/spm/
https://github.com/jaegertracing/jaeger/blob/main/docker-compose/monitor/docker-compose.yml

**Comment by mx-psi** [comment]
@bangbaew Can you file a separate issue for this? Thanks!

**Comment by bangbaew** [comment]
> @bangbaew Can you file a separate issue for this? Thanks!

Done! #26712

**Pull request merged**